modules/apps/29-fee/keeper/escrow.go

```diff
@@ -165,6 +166,7 @@ func (k Keeper) distributeFee(ctx sdk.Context, receiver, refundAccAddress sdk.Ac
 		// then attempt to refund the fee to the original sender
 		err := k.bankKeeper.SendCoinsFromModuleToAccount(cacheCtx, types.ModuleName, refundAccAddress, fee)
 		if err != nil {
+			k.Logger(ctx).Error("error refunding fee to the original sender", "refund address", refundAccAddress, "fee", fee)
```

Comment:
I think we should log the first failed distribution. I think it is very unlikely refunding the fee fails, but trying to distribute to a relayer could definitely fail. It'd be useful for that relayer to notice and see that the fee got refunded after the failed distribution